controlplane/clickhouse/migrations/20230910200148_traces_by_client_mv.sql

```diff
@@ -0,0 +1,50 @@
+-- migrate:up
+
+CREATE MATERIALIZED VIEW IF NOT EXISTS cosmo.traces_by_client_quarter_hourly_mv (
+    Timestamp DateTime('UTC') CODEC (Delta(4), ZSTD(1)),
+    ClientName String CODEC (ZSTD(1)),
+    ClientVersion String CODEC(ZSTD(1)),
+    OrganizationID String CODEC(ZSTD(1)),
+    FederatedGraphID String CODEC(ZSTD(1)),
+    TotalRequests UInt64 CODEC(ZSTD(1)),
+    TotalRequestsError UInt64 CODEC(ZSTD(1)),
+    TotalRequestsOk UInt64 CODEC(ZSTD(1)),
+    DurationQuantiles AggregateFunction(quantiles(0.75, 0.9, 0.95, 0.99), Int64) CODEC(ZSTD(1)),
+    LastCalled DateTime('UTC') CODEC (ZSTD(1))
+)
+ENGINE = SummingMergeTree
+PARTITION BY toDate(Timestamp)
+ORDER BY (
+    toUnixTimestamp(Timestamp), FederatedGraphID, OrganizationID, ClientName
```

Comment:
If the ClientName is the "primary" key put it as the second order by key.

Reply:
Sticking with this order for this one